GPU/GPUTracking/DataTypes/GPUTRDTrack.h

```diff
@@ -86,6 +86,7 @@ class GPUTRDTrack_t : public T
   GPUd() short getCollisionId() const { return mCollisionId; }
   GPUd() int getNtracklets() const;
   GPUd() float getChi2() const { return mChi2; }
+  GPUd() unsigned char getIsPadrowCrossing() const { return mIsPadrowCrossing; }
```

Comment:
Maybe you could also add a getter `bool getIsPadrowCrossing(int layer)` for convenience?